.isort.cfg

```diff
@@ -0,0 +1,10 @@
+[settings]
```

Comment:
What does this config file control?

Reply:
The isort tool -- it was setup automatically by `pyscaffold`. I think it is kind of like black for your imports.